packages/ui/src/validators/modals/validatorCard/ValidatorDetail.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ButtonPrimary } from '@/common/components/buttons'
+import { MarkdownPreview } from '@/common/components/MarkdownPreview'
+import { ModalFooter } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { SidePaneBody, SidePaneLabel, SidePaneRow, SidePaneText } from '@/common/components/SidePane'
+import { NumericValueStat, StatisticsThreeColumns } from '@/common/components/statistics'
+import { TextSmall } from '@/common/components/typography'
+
+import { Validator } from '../../types'
+
+interface Props {
+  validator: Validator
+}
+
+export const ValidatorDetail = ({ validator }: Props) => {
+  return (
+    <>
+      <SidePaneBody>
+        <Details gap={24}>
+          <RowGapBlock gap={4}>
+            <h6>Key elements</h6>
+            <ModalStatistics>
```

Comment:
Also please match the [design](https://www.figma.com/file/QnjZuWYtGimTEkQlN6MNlH/Validators?type=design&node-id=1799-410079&mode=design&t=CTRyskvdyi8y9VYk-0) order for the statistics.

![image](https://github.com/Joystream/pioneer/assets/6571453/cee8226a-3b73-4630-b76d-5a5b732f4d26)